templates/deployment.yaml

```diff
@@ -142,13 +183,54 @@ spec:
           value: "{{ .Values.ipsec.remote_ping_endpoint }}"
         - name: LOCAL_PING_ENDPOINT
           value: "{{ .Values.ipsec.local_ping_endpoint }}"
+      ## VXLAN Connection ##
+      {{- if .Values.vxlan.enabled }}
+      {{- range $index, $element := .Values.vxlan.connectors }}
+      - name: vxlan-connector-{{ $element.name }}
+        image: travelping/vxlan-connector
+        imagePullPolicy: Always
+        securityContext:
+          capabilities:
+            add:
+              - NET_ADMIN
+        env:
+        - name: PEER
+          value: "{{ $element.peer }}"
+        {{- if $element.bridge }}
+        - name: BRIDGE
+          value: "{{ $element.bridge }}"
+        {{- end }}
+        - name: BRIDGED_IFACES
```

Comment:
see readme: plural name vs. singular value

Reply:
multiple values are supported in a space separted list, eg. "eth1 eth2 net0". I clearified this in the README.